modules/registry-accessing-directly.adoc

```diff
@@ -1,70 +1,51 @@
 // Module included in the following assemblies:
 //
-// * assembly/registry
+// * registry/accessing-the-registry.adoc
 
 [id="registry-accessing-directly_{context}"]
-= Accessing registry directly
+= Accessing registry directly from the cluster
 
-You can access the registry directly to invoke `podman` commands. This allows
-you to push images to or pull them from the integrated registry directly using
-operations like `podman-push` or `podman-pull`. To do so, you must be logged in
-to the registry using the `podman-login` command. The operations you can perform
-depend on your user permissions, as described in the following sections.
+You can access the registry from inside the cluster.
 
-.Prerequisites
+.Procedure
 
-* For any direct access, you must have a regular user for your preferred identity
-provider.
-** A regular user can generate an access token required for logging in to
-the registry.
-** System users, such as *system:admin*, cannot obtain access tokens
-and, therefore, cannot access the registry directly.
-+
-For example, if you are using `HTPASSWD` authentication, you can create one
-using the following command:
-+
-----
-# htpasswd /etc/origin/openshift-htpasswd <user_name>
-----
+Access the registry from the cluster by using internal routes:
 
-* For pulling images, for example when using the `podman-pull` command,
-the user must have the *registry-viewer* role. To add this role:
+. Access the node:
+.. Get the node's address:
 +
 ----
-$ oc policy add-role-to-user registry-viewer <user_name>
+$ oc get nodes
+$ oc debug nodes/<node_address>
 ----
-
-* For writing or pushing images, for example when using the `podman-push` command,
-the user must have the *registry-editor* role. To add this role:
++
+. Log in to the container image registry by using your access token:
 +
 ----
-$ oc policy add-role-to-user registry-editor <user_name>
+$ oc login -u kubeadmin -p <PASSWORD_FROM_INSTALL_LOG>
 ----
-
-.Procedure
-
-. Log in to the registry directly:
-
-.. Ensure you are logged in to {product-title} as a *regular user*:
 +
 ----
-$ oc login
+$ podman login -u kubeadmin -p $(oc whoami -t) image-registry.openshift-image-registry.svc:5000
 ----
-
-.. Log in to the container image registry by using your access token:
++
+You should see a message confirming login, such as:
 +
 ----
-$ podman-login -u openshift -p $(oc whoami -t) <registry_ip>:<port>
+Login Succeeded!
 ----
 +
 [NOTE]
 ====
 You can pass any value for the username, the token contains all necessary
```

Comment:
s/username/user name/

Reply:
x2 in this para.

Reply:
Also maybe semicolon instead: "You can pass any value for the user name; the token contains all necessary information."